src/components/TaxonDisplay.tsx

```diff
@@ -1,7 +1,21 @@
 //IMPORT REACT packages
 import React from "react";
 //IMPORT MUI packages
-import { Card, CardContent, Typography } from "@mui/material";
+import { Card, CardContent, Typography, Theme } from "@mui/material";
+import { makeStyles } from "@mui/styles";
+
+/*
+ * STYLE definitions for useStyles hook
+ */
+const useStyles = makeStyles((globalTheme: Theme) => ({
+  containerClass: {},
+  taxonNameClass: {
+    color: globalTheme.palette.secondary.main,
+  },
+  taxonIdClass: {
+    color: globalTheme.palette.secondary.main,
```

Comment:
Probably only need taxonNameClass or taxonIdClass here, unless you intend to make these different soon in which case disregard